repo_structure/repolint.json

```diff
@@ -86,7 +86,19 @@
       "source-license-headers-exist:file-starts-with": [
         "warning",
         {
-          "files": ["**/*.js", "!**/node_modules/**", "**/*.go", "!**/vendor/**", "!**/*.pb.go", "!**/*.gen.go", "!**/third_party/**"],
+          "files": [
+            "**/*.js",
+            "**/*.ts",
+            "**/*.go",
+            "**/*.java",
+            "**/*.yaml",
```

Comment:
Do we need to consider `yaml`?
If yes, maybe there needs a `.yml` as another possible extension.

Reply:
It is a bit uncertain how far we need to go into requiring this on "all files". When the requirement first came in it was argued that absolutely every file needed the header but this just didn't make sense and so it was softened a bit. I'm fine with leaving them out.